packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsHeader.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import { useHistory } from 'react-router-dom';
 import CloseIcon from '@/assets/svgs/ic-close-dark.svg';
+import { useRegulationFlags } from '@/hooks';
+import { useActiveTradingAccount } from '@deriv/api';
 import { Text } from '@deriv-com/ui';
 
-type TCompareAccountsHeader = {
-    isDemo?: boolean;
-    isEuRegion?: boolean;
-};
-
-const CompareAccountsHeader = ({ isDemo, isEuRegion }: TCompareAccountsHeader) => {
+const CompareAccountsHeader = () => {
     const history = useHistory();
+    const { data: activeTrading } = useActiveTradingAccount();
```

Comment:
let's add deriv when we want to destructure it from now on to prevent misunderstanding
```suggestion
    const { data: activeDerivTrading } = useActiveTradingAccount();
```